ingest/muxer.go

```diff
@@ -1438,7 +1439,28 @@ func isFatalConnError(err error) bool {
 	return false
 }
 
+func (im *IngestMuxer) quitableSleep(dur time.Duration) (quit bool) {
+	select {
+	case _ = <-time.After(defaultRetryTime):
```

Comment:
should this be using dur instead of the default constant?

Reply:
OOF.  good catch!